controllers/glance_controller.go

```diff
@@ -166,7 +165,7 @@ func (r *GlanceReconciler) Reconcile(ctx context.Context, req ctrl.Request) (res
 		instance.Status.Conditions.Init(&cl)
 
 		// Register overall status immediately to have an early feedback e.g. in the cli
-		return ctrl.Result{}, nil
+		return ctrl.Result{}, err
```

Comment:
I suspect this is a leftover of my troubleshooting during the development. I'm going to restore this because I'm not going to introduce any change in the patter used across the operator. Thanks, this is a good catch and let me know if you find any other "violation" of common patters that I might avoid in this context. Restoring it in my next PS.